reporters_db/data/laws.json

```diff
@@ -5029,6 +5114,40 @@
             "variations": []
         }
     ],
+    "S. Exec. Doc.": [
+        {
+            "cite_type": "treaty",
+            "end": "1980-12-31T23:59:59",
+            "examples": [
+                "S. Exec. Doc. No. 38"
+            ],
+            "href": "https://law.resource.org/pub/us/code/blue/IndigoBook.html#T6",
+            "jurisdiction": "United States",
+            "name": "Senate Executive Documents",
+            "regexes": [
+                "$reporter\\s*(?:No\\.|Number)?\\s*(?P<number>\\d*[A-Z]?)"
+            ],
+            "start": "1778-01-01T00:00:00",
+            "variations": []
+        }
+    ],
+    "S. Treaty Doc.": [
+        {
+            "cite_type": "treaty",
+            "end": null,
+            "examples": [
+                "S. Treaty Doc. No. 100-20"
```

Comment:
What does the example mean... 100-20 specifically.  It's classified as a number ... but how does it work?

Reply:
The first number refers to the Congress, and the second number is sequential for each document in that Congress. So the example is the Convention Against Torture, which was the 20th treaty document considered by the Senate during the 100th Congress. They used to have a different numbering convention which I have not added retroactive support for yet, but I can do that.

I would also be happy to try to find some way to capture that data with fields like Congress number and sequential number, but we won't have that information for treaties from other sources so it would probably not be worthwhile. I know Eyecite has a mechanism for guessing the year based on the reporter volume, and we could do something similar using the Congress number if you think that is worthwhile.

Reply:
Thank you for explaining it to me.